tekton/release-cheat-sheet.md

```diff
@@ -130,6 +130,24 @@ the pipelines repo, a terminal window and a text editor.
 1. Update [the catalog repo](https://github.com/tektoncd/catalog) test infrastructure
 to use the new release by updating the `RELEASE_YAML` link in [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/master/test/e2e-tests.sh).
 
+1. Update the docs on the website with the latest version released:
+
+    ```bash
+    # From the website repo root, clean and up to date
+    # Update the config
+    cd sync
+    python3 -m venv .venv
+    source .venv/bin/activate    
+    pip3 install -r requirements.txt
+    ./versions.py add -p pipeline [version]
```

Comment:
version is equivalent to TEKTON_VERSION i.e. `v0.20.0`?

Reply:
`Error: no such option: -p` is it `--project`?

Reply:
Yeah, if you do ./versions.py --help

```bash
python3 versions.py --help     
Usage: versions.py [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  add  add a new version in the config for the specified project
  rm   remove a version from the config for the specified project
```
The flag is not there. Looks, there the doc needs to be updated as well.

As for the the command needed. This worked just fine for me.
```bash
python3 versions.py add --project pipeline v0.20.0
```

Reply:
yeah, -p isnt in the docs of the repo. I think its just a typo. It should be `--project`

Reply:
Nice catch, thank you, I will fix this